guides/common/modules/ref_inclusion-of-remote-scap-resources.adoc

```diff
@@ -16,7 +16,11 @@ which is referenced from datastream
 
 By default, the SCAP client is configured to ignore the remote resources and skip the XCCDF rules that rely on the resources.
 The skipped rules then result in the `notchecked` status.
-In a connected network environment, you can enable the download of remote resources on hosts in {Project}.
+
+For hosts with internet access, you can enable the download of remote resources on hosts in {Project}.
```

Comment:
How would you do that/is this documented or maybe just the default for Ansible/Puppet? Or maybe reword to "...., your chosen OpenSCAP deployment method will automatically download SCAP content from the internet"?

Reply:
That isn't accurate. It doesn't depend on the deployment method. The SCAP scanner (oscap) on the host downloads the remote SCAP content.

Reply:
OK, thanks for clarifying!